README.rst

```diff
@@ -6,363 +6,106 @@ Additional modules for the Cyclus nuclear fuel cycle simulator from the
 University of Wisconsin - Madison are intended to be support innovative
 fuel cycle simulations with the Cyclus fuel cycle simulator.
 
-To see user and developer documentation for this code, please visit
-the `Cyclus Homepage`_.
+Cycamore require Cyclus. If you don't have installed Cyclus yet please checkout
```

Comment:
(The second bullet point really highlights how hard it is to master English. It had me shaking my head....)